### DISCUSSION TITLE
Integrating with the notion API

### DISCUSSION THREAD

**Discussion by rollonbears234** [paperboi/kindle2notion#49] [Ideas]
Love the idea! 

Would it be a simple change to instead of using the `token_v2` cookie to instead use an API key? I'd rather not provide my auth token to the script when instead I can create an API key just for this use case. 

Also, it seems possible to access the highlights from the kindle app instead of having to plug in our device. Was this something that was considered, was there a technical limitation, or did you cut the scope of the initial launch to not include this?

**Comment by paperboi** [comment]
It's in the roadmap... I am still learning more on how to use the API.

**Comment by paperboi** [comment]
Regarding your second question, the Kindle app doesn't account for books from third-party sources i.e. books not purchased directly from the Amazon store. The clippings from these books are stored within the Kindle device itself.
